default/species.txt

```diff
@@ -1542,6 +1560,25 @@ Species
 //####                                               ####\\
 //#######################################################\\
 
+// BROAD_EP and NARROW_EP are stub placeholders for applying the
+// board/narrow environment tolerance description to a species.
+BROAD_EP
+'''
+        EffectsGroup
+            description = "BROAD_EP"
+            scope = Source
+            activation = None
+            effects = SetPlanetType type = Target.OriginalType
```

Comment:
It doesn't hold up this pull request, but I don't like this important-looking effect in an effectsgroup where it's never intended to function. There should probably be a Dummy or DoNothing effect for this purpose...